<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Obsession Hotline Client Alpha</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionNew_Window"/>
    <addaction name="actionOpen_Connection"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionTrackers"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Tab"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Contra"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionBookmarks"/>
    <addaction name="actionIcons"/>
   </widget>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="actionChat"/>
    <addaction name="actionServer_Agreement"/>
    <addaction name="actionFiles"/>
    <addaction name="actionLinear_News"/>
    <addaction name="actionThreaded_News"/>
    <addaction name="separator"/>
    <addaction name="actionDownloads"/>
    <addaction name="separator"/>
    <addaction name="actionBroadcast"/>
    <addaction name="separator"/>
    <addaction name="actionView_account"/>
    <addaction name="actionNew_account"/>
   </widget>
   <addaction name="menuConnection"/>
   <addaction name="menuServer"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_Connection">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/main/interfaceIcons/mainConnect.png</normaloff>:/main/interfaceIcons/mainConnect.png</iconset>
   </property>
   <property name="text">
    <string>Open Connection...</string>
   </property>
   <property name="toolTip">
    <string>Open a new connection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/main/interfaceIcons/mainClose.png</normaloff>:/main/interfaceIcons/mainClose.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionServer_Info">
   <property name="text">
    <string>Server Info</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/main/interfaceIcons/mainDisconnect.png</normaloff>:/main/interfaceIcons/mainDisconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect...</string>
   </property>
  </action>
  <action name="actionAbout_Contra">
   <property name="text">
    <string>About Obsession...</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionServer_Agreement">
   <property name="text">
    <string>Agreement</string>
   </property>
  </action>
  <action name="actionTrackers">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/main/interfaceIcons/mainTrackers.png</normaloff>:/main/interfaceIcons/mainTrackers.png</iconset>
   </property>
   <property name="text">
    <string>Trackers...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/main/interfaceIcons/mainNew.png</normaloff>:/main/interfaceIcons/mainNew.png</iconset>
   </property>
   <property name="text">
    <string>New Window...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionFiles">
   <property name="text">
    <string>Files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionIcons">
   <property name="text">
    <string>Icons...</string>
   </property>
  </action>
  <action name="actionLinear_News">
   <property name="text">
    <string>Message Board</string>
   </property>
  </action>
  <action name="actionDownloads">
   <property name="text">
    <string>Downloads</string>
   </property>
  </action>
  <action name="actionClose_Tab">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Close Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionThreaded_News">
   <property name="text">
    <string>News</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionDebug_console">
   <property name="text">
    <string>Debug console</string>
   </property>
  </action>
  <action name="actionBookmarks">
   <property name="text">
    <string>Bookmarks...</string>
   </property>
  </action>
  <action name="actionChange_own_password">
   <property name="text">
    <string>Change own password...</string>
   </property>
  </action>
  <action name="actionView_account">
   <property name="text">
    <string>View account...</string>
   </property>
  </action>
  <action name="actionNew_account">
   <property name="text">
    <string>New account...</string>
   </property>
  </action>
  <action name="actionBroadcast">
   <property name="text">
    <string>Broadcast..</string>
   </property>
  </action>
  <action name="actionChat">
   <property name="text">
    <string>Chat</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
